rest/persistent_config_test.go

```diff
@@ -389,7 +389,7 @@ func TestImportFilterEndpoint(t *testing.T) {
 	resp.RequireStatus(http.StatusOK)
 
 	// Modify the import filter to always reject import
-	resp = BootstrapAdminRequest(t, http.MethodPut, "/db1/_config/import_filter", `function(){return false}`)
+	resp = BootstrapAdminRequest(t, http.MethodPut, "/db1/_config/import_filter", `function(doc){return false}`)
```

Comment:
is it breaking backward compatibility to require 1 argument? presumably we previously let 0 argument functions through?

A great feature of JS I suppose.

We _definitely_ support setting only one or two args for sync functions for backward compatibility and I think the the validation in `rest` package breaks that.

Reply:
A zero-arg import filter would be useless, wouldn't it? (If you always want to reject or approve import, you don't need a filter at all.)

Reply:
True! This test looks right and you took care of the validation for one argument which I think is correct.